docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -128,6 +128,12 @@ specific environment, you may need to choose different CIDRs.
 So, once you have decided which type of network to create, following the
 appropriate instructions for one of *a)*, *b)*, *c)* or *d)*.
 
+For AWS, chose a) or c), and Disable Source/Dest. checks on your instances with
+the following EC2 CLI command or by right clicking the Instance, and selecting
+it from the Networking submenu.
```

Comment:
Maybe use the specific name of the option to click instead of "it"? It's not called Disable Source/Dest. here, I think it's `Change Source/Dest. Check`.